src/Teams/Controllers/ManageTasksController.cs

```diff
@@ -105,7 +106,7 @@ public async Task<IActionResult> EditTaskAsync(int teamId, int taskId, string er
             var task = await _manageTasksService.GetTaskByIdAsync(taskId);
             var teamMembers = await GetAllTeamMembersAsync(teamId);
 
-            EditTaskViewModel model = new EditTaskViewModel
+            TaskFormViewModel model = new TaskFormViewModel
```

Comment:
use var here

Reply:
why it was renamed?

Reply:
because I use this form not only for editing but also for adding and name EditTaskViewModel does not fit the meaning